README.md

```diff
@@ -1,15 +1,193 @@
 
 # Library Management System
 
-This is a Library Management System built with Spring Boot.
+This is a Spring Boot application that implements books, authors, users, tasks etc. that are gathered in a MySQL database. Each role has a different degree of access to the application.
+ 
 
-### Start application
+### How do I start the application?
 
 ```
 docker-compose up --build
 ```
+###What features are implemented?
+- Database relations
+- Dockerfile/Docker compose
+- Create, Read, Update and Delete operations
+- Customized exceptions
+- Spring Boot Security 
+- RabbitMQ and JMS
+- Integration Testing
+- Thymeleaf 
 
+Additional features: 
+MySQL database 
 
+###How do I deploy it ?
+###Option 1
 
+Clone the repository:
+
+- Navigate to the folder where you want the application to be saved.
+
+- Use your console to run the following:
+
+   git clone https://github.com/fireblazrs/library-management-system
+
+###Or
+
```

Comment:
```suggestion

```

It is preferable if a user downloads a the packaged application (as above) or an image of our application (as below). Since they are accessing our project repository on Github, it is reasonable to assume they know how to use `git clone`.